src/techui_builder/generate.py

```diff
@@ -306,10 +310,8 @@ def build_groups(self):
 
         self.group.version("2.0.0")
         self.group.add_widget(widgets)
-
-    def write_screen(self):
-        # Add the created groups to the screen and write the screen
         self.screen_.add_widget(self.group)
-        self.screen_.write_screen(
-            "./example-synoptic/" + self.screen_name + ".bob"
-        )  # TODO: Make this into a directory without example synoptic
+
+    def write_screen(self, directory: Path):
+        """Write the screen to file"""
+        self.screen_.write_screen(f"{directory}/{self.screen_name}.bob")
```

Comment:
If using a Path for `directory`, you can use the `joinpath()` class method to combine them, anf this keeps it as a Path object
```suggestion
        self.screen_.write_screen(directory.joinpath(f"{self.screen_name}.bob"))
```

Reply:
self.screen_.write_screen accepts type string, not path unfortunately.

Reply:
is `_write_screen()` a `phoebusgen` method? If so then that's a shame...

Reply:
No worries then!